include/NovelRT.Interop/Animation/NovelRTSpriteAnimatorState.h

```diff
@@ -0,0 +1,41 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#include <stddef.h>
```

Comment:
uhhhhhhhh remind me what this header does?

Reply:
As discussed, this is needed to define `size_t` in Linux builds